include/llvm/CodeGen/AsmPrinter.h

```diff
@@ -112,6 +112,9 @@ class AsmPrinter : public MachineFunctionPass {
   typedef std::pair<const GlobalVariable *, unsigned> GOTEquivUsePair;
   MapVector<const MCSymbol *, GOTEquivUsePair> GlobalGOTEquivs;
 
+  /// Sections that need to be referenced in the PGLT
+  std::vector<const MCSection*> PGLT;
+
```

Comment:
Maybe initialize PGLT with default #0 bin, to align outside view (i.e., `GetPGLTIndex`) with internal representation.